packages/zowe-explorer-api/src/vscode/ui/utils/HTMLTemplate.ts

```diff
@@ -14,28 +14,24 @@
 /**
  * HTML template that is compiled with Mustache to load a WebView instance at runtime.
  */
-const HTMLTemplate: string = `
+const HTMLTemplate: string = /*html*/ `
 <!DOCTYPE html>
 <html lang="en">
     <head>
         <meta charset="UTF-8">
         <title>{{ title }}</title>
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta 
-            http-equiv="Content-Security-Policy" 
-            content="default-src 'none'; font-src data: {{ cspSource }}; img-src data: vscode-resource: https:; script-src {{#unsafeEval}}'unsafe-eval'{{/unsafeEval}} 'nonce-{{ nonce }}';
-            style-src {{ cspSource }} vscode-resource: 'unsafe-inline' http: https: data:;"
+        <meta
+            http-equiv="Content-Security-Policy"
+            content="default-src 'none'; font-src data: https:; img-src data: vscode-resource: https:; script-src 'nonce-{{ nonce }}';
+            style-src vscode-resource: 'unsafe-inline' http: https: data:;"
         />
         <base href="{{ uris.resource.build }}">
-        {{#uris.resource.css}}
-        <link type="text/css" rel="stylesheet" href="{{ uris.resource.css }}" />
-        {{/uris.resource.css}}
-        {{#uris.resource.codicons}}
-        <link type="text/css" rel="stylesheet" href="{{ uris.resource.codicons }}" />
-        {{/uris.resource.codicons}}
+        {{{ style }}}
```

Comment:
also, we might want to integrate the styles with the `#uris.resource.css` instead of overriding with the generic `{{{ style }}}` block 😋

Reply:
let's leave this for when we move to integrated terminals
- #3079

Reply:
I believe the webview is in Ok shape as-is on the main branch 🙏 
Resolving!